phys2bids/interfaces/mat.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""phys2bids interface for MATLAB files."""
+
+import logging
+
+import numpy as np
+from scipy.io import loadmat
+
+from phys2bids.physio_obj import BlueprintInput
+
+LGR = logging.getLogger(__name__)
+
+
+def populate_phys_input(filename, chtrig):
+    """
+    Populate object phys_input from MATLAB files.
+
+    Parameters
+    ----------
+    filename: str
+        path to the txt labchart file
+    chtrig : int
+        index of trigger channel.
+        !!! ATTENTION: IT'S MEANT TO REPRESENT AN INDEX STARTING FROM 1 !!!
+
+    Returns
+    -------
+    BlueprintInput
+
+    Note
+    ----
+    chtrig is not a 0-based Python index - instead, it's human readable (i.e., 1-based).
+    This is handy because, when initialising the class, a new channel corresponding
+    to time is added at the beginning - that is already taken into account!
+
+    See Also
+    --------
+    physio_obj.BlueprintInput
+    """
+    # Load MATLAB file into dictionary.
+    mat_dict = loadmat(filename)
```

Comment:
Ok I have quite a bad news here.
look at the `loadmat` Notes. Matlab files v7.3 aren't supported by it - instead, you need [h5py](https://www.h5py.org/) to [read those](https://stackoverflow.com/a/19340117). Alternatively you can break the code in case a matlab v7.3 file is given as input saying that we don't support it (yet).
In both cases, I would try this import and if it fails act in consequence (either trying using hdf5, or saying we don't support the format, or (better) both.